memiavl/db.go

```diff
@@ -217,7 +220,7 @@ func (db *DB) RewriteSnapshotBackground() error {
 			return
 		}
 
-		ch <- snapshotResult{mtree: mtree}
+		ch <- snapshotResult{mtree: mtree, path: path}
```

Comment:
An alternative is simply return the snapshot version here.

Reply:
done, but I think file path seems more related